.pre-commit-config.yaml

```diff
@@ -2,20 +2,14 @@ default_language_version:
   python: python3.8
 
 repos:
-  - repo: https://github.com/psf/black
-    rev: 22.6.0
+  - repo: local
     hooks:
-    - id: black
-
-  - repo: https://github.com/PyCQA/flake8
-    rev: 4.0.1
-    hooks:
-    - id: flake8
-
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
-    hooks:
-    - id: isort
+    - id: check
+      name: check
+      entry: just check
+      language: system
+      types: [python]
+      require_serial: true
```

Comment:
The `require_serial` isn't strictly needed since there's just the one local hook, but I'm leaving it in for when we come to changing to ruff